### PR TITLE
Removed all instances of velocity being defined for `CharacterBody2D`

### LINKED ISSUE #6860: Member "velocity" redefined (original in native class 'CharacterBody2D')
**Your Godot version:**
4.0

**Issue description:**
Error "Member "velocity" redefined (original in native class 'CharacterBody2D')" when copying the snippet for `move_and_slide` at the bottom of the page.

```
extends CharacterBody2D

var run_speed = 350
var jump_speed = -1000
var gravity = 2500

var velocity = Vector2()

func get_input():
    velocity.x = 0
    var right = Input.is_action_pressed('ui_right')
    var left = Input.is_action_pressed('ui_left')
    var jump = Input.is_action_just_pressed('ui_select')

    if is_on_floor() and jump:
        velocity.y = jump_speed
    if right:
        velocity.x += run_speed
    if left:
        velocity.x -= run_speed

func _physics_process(delta):
    velocity.y += gravity * delta
    get_input()
    velocity = move_and_slide(velocity, Vector2(0, -1))
```

**URL to the documentation page:**
https://docs.godotengine.org/en/stable/tutorials/physics/physics_introduction.html#move-and-slide

If you know how to fix the issue you are reporting please
consider opening a pull request. We provide a tutorial on
using git here: https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html,
writing documentation at https://docs.godotengine.org/en/stable/community/contributing/docs_writing_guidelines.html
and contributing to the class reference here: https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html

### PR DESCRIPTION
**Pull request by ShatReal**
`velocity` is a built-in property of `CharacterBody2D`. There is no need to define it, or use it in move_and_slide(). There is also no need to pass in up direction since that is a built-in property too.

Closes #6860

### CODE REVIEW

**Review by fabriceci — APPROVED** [review]
LGTM

File: tutorials/physics/kinematic_character_2d.rst [hunk 1]
```diff
@@ -201,11 +197,8 @@ This adds basic support for walking when pressing left and right:
         else:
             velocity.x = 0
 
-        # We don't need to multiply velocity by delta because "move_and_slide" already takes delta time into account.
-
-        # The second parameter of "move_and_slide" is the normal pointing up.
-        # In the case of a 2D platformer, in Godot, upward is negative y, which translates to -1 as a normal.
-        move_and_slide(velocity, Vector2(0, -1))
+        # "move_and_slide" takes delta time into account.
```
Comment (fabriceci) [inline tutorials/physics/kinematic_character_2d.rst:200]: **already** takes delta, like c#

Comment (ShatReal) [reply tutorials/physics/kinematic_character_2d.rst]: ok fixed

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks for contributing, merged! Will show up on `stable` once we cherrypick it. And thanks for the review!

**Pull request merged**